test/e2e/framework.go

```diff
@@ -200,6 +214,14 @@ func (f *Framework) afterEach() {
 	f.Client = nil
 }
 
+func (f *Framework) CreateNamespace(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error) {
+	ns, err := createTestingNS(baseName, c, labels)
```

Comment:
Seems like `createTestingNS` should just be pulled into `framework.go` and de-duped with this function.